### ISSUE TITLE
Compatibily Fifa 20

### ISSUE THREAD

**Issue by totti240282** [trydis/FIFA-Ultimate-Team-Toolkit#418]
Hello,
this library is ready for Fifa 20 ?

**Comment by marius070490** [comment]
Hello, 

Did you manage to have it working?
I see one major problem, that I have not ideea how to fix. 

EA has recently added to their authentication metod (the one when you get you session guid) a new field "ds" in the payload, which seems to be some sort of hash. They generate a new code for each login request. I took a look to the WebApp minified JS, and I discovered they generate the code from JS ('ds': this[_0x1594('0x903')]), but the code is very hard to follow, as is minified.

Another problem I found, which I am trying to solve now, is that the OTP code is not sent to my email, although the server response seems to be fine.

Is anyone else working on these issues?
